flagsmith/flagsmith.py

```diff
@@ -51,11 +56,13 @@ def __init__(
         proxies: typing.Dict[str, str] = None,
         offline_mode: bool = False,
         offline_handler: BaseOfflineHandler = None,
+        use_stream: bool = False,
```

Comment:
Can we change this to `enable_realtime_updates` (just to be consistent with other SDKs)?